plat/mediatek/mt8183/platform.mk

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+MTK_PLAT      := plat/mediatek
+MTK_PLAT_SOC  := ${MTK_PLAT}/${PLAT}
+
+PLAT_INCLUDES := -I${MTK_PLAT}/common/                            \
+                 -I${MTK_PLAT}/common/drivers/uart/               \
+                 -I${MTK_PLAT_SOC}/drivers/                       \
+                 -I${MTK_PLAT_SOC}/drivers/spmc/                  \
+                 -I${MTK_PLAT_SOC}/drivers/gpio/                  \
+                 -I${MTK_PLAT_SOC}/drivers/pmic/                  \
+                 -I${MTK_PLAT_SOC}/drivers/rtc/                   \
+                 -I${MTK_PLAT_SOC}/include/                       \
+                 -Iinclude/common/                                \
+                 -Iinclude/drivers/                               \
+                 -Idrivers/arm/gic/v3/
+
+PLAT_BL_COMMON_SOURCES := lib/xlat_tables/aarch64/xlat_tables.c	      \
+                          lib/xlat_tables/xlat_tables_common.c        \
+                          plat/common/plat_gicv2.c                    \
+                          plat/common/aarch64/platform_helpers.S      \
```

Comment:
Remove ``plat/common/aarch64/platform_helpers.S`` from here. It's already added from the main Makefile, and it makes make give the following warnings:
```
Makefile:776: warning: overriding recipe for target 'build/mt8183/debug/bl31/platform_helpers.o'
Makefile:776: warning: ignoring old recipe for target 'build/mt8183/debug/bl31/platform_helpers.o'
```